langchain/src/chat_models/base.ts

```diff
@@ -38,28 +42,31 @@ export abstract class BaseChatModel extends BaseLanguageModel {
 
   async generate(
     messages: BaseChatMessage[][],
-    stop?: string[]
+    stop?: string[],
+    callbackManager?: CallbackManager
   ): Promise<LLMResult> {
     const generations: ChatGeneration[][] = [];
     const llmOutputs: LLMResult["llmOutput"][] = [];
     const messageStrings: string[] = messages.map((messageList) =>
       getBufferString(messageList)
     );
-    await this.callbackManager.handleLLMStart(
+    const callbackManager_ = await this.configureCallbackManager(
```

Comment:
I still think this should move to CallbackManager? (I get this is accessing properties on instance, but those can be passed in?